include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_dual_splitting.cpp

```diff
@@ -644,7 +653,10 @@ TimeIntBDFDualSplitting<dim, Number>::projection_step()
     if(this->print_solver_info() and not(this->is_test))
     {
       this->pcout << std::endl << "Explicit projection step:";
-      print_wall_time(this->pcout, timer.wall_time());
+      if(this->param.spatial_discretization == SpatialDiscretization::HDIV)
+        print_wall_time_with_mass_iter(this->pcout, n_iter_mass, timer.wall_time());
```

Comment:
Why a new function? Can't one use one of the existing functions? See other steps of e.g. the dual splitting scheme where we print both wall times and iteration counts. Such a function should be re-usable here I think.

Reply:
I can't really remember the reason.. :)